bash_scripting_101/bash_scripting_101.md

```diff
@@ -157,21 +172,21 @@ In our installation section, we covered how accessing CLI tools to run bash scri
 
   ![Head output of slightly less basic python script that organizes a picnic gathering menu amongst three friends.](media/head_example_small.png)
 
-### Knowledge Check 2
+### Quiz: Bash commands
 
 Select the three functions of the `cat` command?
 
 [[X]] View
 [[ ]] Categorize
 [[X]] Concatenate
 [[X]] Create New Version
-***********************************************************************
+***
 <div class = "answer">
 
 The `sort` command is used to categorize files in bash.
 
 </div>
-***********************************************************************
+***
 
 ### Syntax
```

Comment:
Examples! how do we actually use these? where do they take us/what does it look like when they do what they do?